pkg/extension/sumologicextension/credentials/encrypt_test.go

```diff
@@ -0,0 +1,17 @@
+package credentials
+
+import (
+	"testing"
+)
+
+func TestXXX(t *testing.T) {
+	// l := localFsCredentialsStore{
+	// 	collectorCredentialsDirectory: "~/.sumologic-otel-collector",
+	// }
+	// _ = l
+
+	// cc, err := l.Get("700c22e0935376be9a8452ec8c942a10")
+	// require.NoError(t, err)
+
+	// t.Logf("out: %+v", cc)
+}
```

Comment:
shouldn't be uncommented?

Reply:
derp, this shouldn't land in this PR :) 
removed